packages/api-explorer/src/JsonForm/index.jsx

```diff
@@ -0,0 +1,82 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import JSONEditor from '@json-editor/json-editor'
+
+import getSchemaToRender from './getSchemaToRender'
+import antdTheme from './antd-theme-json-editor'
+import getCustomEditor from './get-custom-editor'
+import arrayCustomEditor from './array-custom-editors'
+import objectCustomEditor from './object-custom-editors'
+
+import './bootstrap4.css'
+import './custom-bootstrap4.css'
+
+function setAllEditors () {
+  const editorsKeys = Object.keys(JSONEditor.defaults.editors)
+  editorsKeys
+    .filter(key => key !== 'array' && key !== 'object').forEach(key => {
+    JSONEditor.defaults.editors[key] = getCustomEditor(key);
+  });
+  JSONEditor.defaults.editors.array = arrayCustomEditor()
+  JSONEditor.defaults.editors.object = objectCustomEditor()
+}
+
+export default class JsonForm extends Component {
+    constructor(props) {
+      super(props);
+      this.editor = null;
+      this.ref = null;
+    }
+  
+    createEditor(element) {
+      const {onChange, schema} = this.props
+      if (this.editor === null) {
+        const schemaToRender = getSchemaToRender(schema)
+        JSONEditor.defaults.themes.antdTheme = antdTheme
```

Comment:
Let's move all `JSONEditor` configurations into the 'setAllEditors` function to help readability.

Also: why don't we call `setAllEditors` something link `configureJSONEditor`?